lib/segment/src/vector_storage/sparse/simple_sparse_vector_storage.rs

```diff
@@ -111,11 +113,15 @@ impl SimpleSparseVectorStorage {
             }
         }
 
+        let key_enc = bincode::serialize(&key).unwrap();
+        let record_enc = bincode::serialize(&record).unwrap();
+
+        hw_counter
+            .vector_io_write_counter()
+            .incr_delta(key_enc.len() + record_enc.len());
```

Comment:
Are we sure we want to count the key here as well? As far as I'm aware we don't do this for dense vectors.

Reply:
We do for dense vectors too: https://github.com/qdrant/qdrant/blob/measure_vector_insert_io_writes/lib/segment/src/vector_storage/dense/simple_dense_vector_storage.rs#L182

I measured it because it is part of what we store and have to write to disk.
Do you think it would be better to not measure it?